modules/nw-multus-bridge-object.adoc

```diff
@@ -23,50 +23,78 @@ plugin:
 
 |`type`
 |`string`
-|
+|`bridge`
```

Comment:
this is duplicated?

Reply:
@mlguerrero12 I am taking this from https://www.cni.dev/plugins/current/main/bridge/ `type (string, required): “bridge”.`

Reply:
I removed bridge to avoid any confusion

Reply:
it should be 

"The name of the CNI plugin to configure: bridge."

like we have in the other plugins

Reply:
Yes just added that for consistency